modules/@apostrophecms/notification/ui/apos/components/AposNotification.vue

```diff
@@ -1,57 +1,30 @@
 <template>
   <div role="alert" :class="classList">
     <span class="apos-notification__indicator">
-      <AposIndicator
-        :icon="iconComponent"
-        class="apos-notification__indicator__icon"
-        :icon-size="notification.icon ? 16 : 12"
-      />
+      <AposIndicator :icon="iconComponent" class="apos-notification__indicator__icon"
```

Comment:
not sure about all the changes in this file

Reply:
Fixed